lib/__tests__/helpers/testUtils.ts

```diff
@@ -1,4 +1,20 @@
 import { mock } from 'bun:test'
+import { install } from '@sinonjs/fake-timers'
+import { afterAll, beforeEach } from 'bun:test'
+
+export function fakeTimers() {
+  const clock = install()
+
+  beforeEach(() => {
+    clock.reset()
+  })
+
+  afterAll(() => {
+    clock.uninstall()
+  })
+
+  return clock
```

Comment:
helper function so we can run tests in virtual time instead of having to wait for time outs